modules/test/playwright/pages/commerce/commerce-order-content-web/commerceLayoutsPage.ts

```diff
@@ -226,6 +234,33 @@ export class CommerceLayoutsPage {
 		this.orderItemCardButton = page
 			.frameLocator('iframe[title="Select"]')
 			.getByRole('button', {name: 'Select Order Items'});
+		this.orderTypeSelector = page.locator('.modal-content');
+
+		// this.orderTypeSelectorFrame = page.frameLocator('iframe >> nth=1');
+
+		this.orderTypeSelectorFrame = page.frameLocator('iframe').last();
```

Comment:
Unneeded as no more an iframe. You can locate the modal straight via `page.locator('#orderTypeSelection')` now